setup.py

```diff
@@ -0,0 +1,50 @@
+import io
+import os
+import re
+
+from setuptools import find_packages
+from setuptools import setup
+
+
+def read(filename):
+    filename = os.path.join(os.path.dirname(__file__), filename)
+    text_type = type(u"")
+    with io.open(filename, mode="r", encoding='utf-8') as fd:
+        return re.sub(text_type(r':[a-z]+:`~?(.*?)`'), text_type(r'``\1``'), fd.read())
+
+
+setup(
+    name="Github-dork",
+    version="0.1.0",
+    url="https://github.com/grofers/github-dorks",
+    license='MIT',
```

Comment:
Are you sure? See this https://github.com/grofers/github-dorks/blob/master/LICENSE

Using Apache License 2.0